managed/services/management/azure_database.go

```diff
@@ -21,7 +21,7 @@ import (
 	"fmt"
 
 	"github.com/AlekSi/pointer"
-	"github.com/Azure/azure-sdk-for-go/services/resourcegraph/mgmt/2019-04-01/resourcegraph"
+	"github.com/Azure/azure-sdk-for-go/services/resourcegraph/mgmt/2019-04-01/resourcegraph" //nolint:staticcheck
```

Comment:
That package is deprecated, here is a ticket to update it: https://github.com/percona/pmm/issues/1250